ra/ra.go

```diff
@@ -1122,15 +1122,26 @@ func (ra *RegistrationAuthorityImpl) issueCertificateInner(
 		OrderID:        &orderIDInt,
 	}
 
+	// wrapError adds a prefix to an error. If the error is a boulder error then
```

Comment:
This is definitely the right thing to do, I've been thinking we may want to just implement this method on `berrors.BoulderError` though so that it can be used elsewhere (which is probably something we'll want to do more down the line).

Perhaps not necessary for this PR in particular but something to think about.